flask_static_digest/__init__.py

```diff
@@ -3,7 +3,7 @@
 
 from urllib.parse import urljoin
 
-from flask import url_for as flask_url_for
+from flask import _request_ctx_stack, url_for as flask_url_for
```

Comment:
Is there any way we can do what we want here without depending on a private function from Flask?

Reply:
[5f3b61e replaced flask._request_ctx_stack.top.request with flask.request](https://github.com/nickjj/flask-static-digest/pull/33/commits/5f3b61e20618e8deb18ac66759094ae94b50cf78)